src/services/message.service.ts

```diff
@@ -31,6 +31,57 @@ export class MessageService {
     this._sendConstructedReport(report, { files: message.attachments.map((e) => e.url) });
   }
 
+  async sendTextMessage(
+    message: IMessage,
+    content: string,
+    argOptions: ITextMessageOptions
+  ): Promise<void> {
+    const options: ITextMessageOptions = {};
+    options.header = argOptions.header ? `${argOptions.header}\n` : '';
+    options.footer = argOptions.footer ? `\n${argOptions.footer}` : '';
+    options.reply = argOptions.reply || false;
+    options.delimiter = argOptions.delimiter === false ? false : argOptions.delimiter || '\n';
+
+    const replyLength = options.reply ? `${message.author},  `.length : 0;
+    const templateLength = replyLength + options.header.length + options.footer.length;
+
+    let messagesToSend: string[] | null = [`${options.header}${content}${options.footer}`];
+
+    if (content.length + templateLength > Constants.MaxMessageLength) {
```

Comment:
Check the negative and return

Reply:
Why?  A really high percentage of the messages that reach this part need to run the code after the if statement either-or.

Reply:
I makes the code more readable
```
if(dont need to edit){
    send original message
    return
}

split up...
```